python/pyspark/ml/tests.py

```diff
@@ -389,6 +389,21 @@ def test_word2vec_param(self):
         # Check windowSize is set properly
         self.assertEqual(model.getWindowSize(), 6)
 
+    def test_findSynonyms(self):
+        sent = ("a b " * 100 + "a c " * 10).split(" ")
+        doc = spark.createDataFrame([(sent,), (sent,)], ["sentence"])
+        word2Vec = Word2Vec(vectorSize=5, seed=42, inputCol="sentence", outputCol="model")
+        model = word2Vec.fit(doc)
+        model.getVectors().show()
+
+        from pyspark.sql.functions import format_number as fmt
+        from pyspark.ml.linalg import Vector
+        from pyspark.ml.linalg import Vectors
+        model.findSynonyms("a", 2).select("word", fmt("similarity", 5).alias("similarity")).show()
+
+        # model.findSynonymsArray(["a"], 2).select("word", fmt("similarity", 5).alias("similarity")).show()
+        model.findSynonymsArray(Vectors.dense("a"), 2).select("word", fmt("similarity", 5).alias("similarity")).show()
```

Comment:
As per style tests this line is too long. Also why are you creating a vector of "a"?